src/common/schema/application.js

```diff
@@ -395,6 +395,9 @@ module.exports = {
     veteranOtherIncome: ['understandsFinancialDisclosure'],
     spouseGrossIncome: ['understandsFinancialDisclosure'],
     spouseNetIncome: ['understandsFinancialDisclosure'],
-    spouseOtherIncome: ['understandsFinancialDisclosure']
+    spouseOtherIncome: ['understandsFinancialDisclosure'],
```

Comment:
I think these dependencies are not implemented properly. Looking at the documentation, it looks like the dependency is listed first and the fields they are dependent on are listed in the array: https://spacetelescope.github.io/understanding-json-schema/reference/object.html#property-dependencies.